views/roleDetails.njk

```diff
@@ -32,6 +32,7 @@
     {% set heading = '<h1 class="govuk-heading-l">' +  title  + '</h1>' %}
     {% set changeRoleNameLink = '<a href="' + maintainUrl + '/' + roleDetails.roleCode  + '/change-role-name" class="govuk-link" data-qa="change-role-name-link">Change <span class="govuk-visually-hidden"> role name</span> </a>' %}
     {% set changeRoleDescriptionLink = '<a href="' + maintainUrl + '/' + roleDetails.roleCode  + '/change-role-description" class="govuk-link" data-qa="change-role-description-link">Change <span class="govuk-visually-hidden"> role description</span> </a>' %}
+    {% set roleDescriptionWithWrapping = '<span class="wrapped"><pre class="govuk-body govuk-!-margin-bottom-0">' + ("" if roleDetails.roleDescription == null else roleDetails.roleDescription) + '</pre></span>' %}
```

Comment:
We should really try to output the `roleDescription` considering that it could possibly contain cross site scripting characters or characters that could cause the html to be malformed.  Will obviously be fine in this case since we know we have validated it, but would be better to be defensive instead